src/models/flinkDatabaseResource.ts

```diff
@@ -1,7 +1,11 @@
+import type { FlinkAIAgent } from "./flinkAiAgent";
 import type { FlinkAIConnection } from "./flinkAiConnection";
 import type { FlinkAIModel } from "./flinkAiModel";
 import type { FlinkAITool } from "./flinkAiTool";
 import type { FlinkUdf } from "./flinkUDF";
 
+/** Union type for all Flink AI resources. */
+export type FlinkAIResource = FlinkAIModel | FlinkAIConnection | FlinkAITool | FlinkAIAgent;
```

Comment:
Moved from `src/viewProviders/multiViewDelegates/flinkAiDelegate.ts` so we didn't have to update the union over there along with the  `FlinkDatabaseResource` union type